tests/integration/test_logs.py

```diff
@@ -83,3 +92,45 @@ def test_list_tags_log_group(self):
         self.logs_client.delete_log_group(
             logGroupName=group
         )
+
+    def test_put_subscribe_policy(self):
+        iam_client = aws_stack.connect_to_service('iam')
+        lambda_client = aws_stack.connect_to_service('lambda')
+        log_client = aws_stack.connect_to_service('logs')
+
+        role_name = 'role-{}'.format(short_uid())
+        assume_policy_document = {'Version': '2012-10-17',
+                                  'Statement': [
+                                      {
+                                          'Action': 'sts:AssumeRole',
+                                          'Principal': {
+                                              'Service': 'lambda.amazonaws.com'
+                                          },
+                                          'Effect': 'Allow',
+                                          'Sid': '',
+                                      }
+                                  ]
+                                  }
+
+        iam_client.create_role(
```

Comment:
Out of curiosity - is this role creation required here, is the test failing without it?